docs/decisions/2022-07-pet-test-fixtures.md

```diff
@@ -0,0 +1,73 @@
+# Allow "pet" test fixtures
```

Comment:
Because this is not just a point in time decision, but an ongoing guideline for related decisions in the future, it would be good to add a reference to this decision record as a design philosophy: https://github.com/GoogleCloudPlatform/emblem/blob/main/CONTRIBUTING.md#design--project-philosophy

Reply:
See #574